README.md

```diff
@@ -376,18 +376,27 @@ This repository contains a curated list of awesome open source libraries that wi
 * [CuML](https://github.com/rapidsai/cuml) ![](https://img.shields.io/github/stars/rapidsai/cuml.svg?style=social) - cuML is a suite of libraries that implement machine learning algorithms and mathematical primitives functions that share compatible APIs with other RAPIDS projects.
 * [CuPy](https://github.com/cupy/cupy) ![](https://img.shields.io/github/stars/cupy/cupy.svg?style=social) - An implementation of NumPy-compatible multi-dimensional array on CUDA. CuPy consists of the core multi-dimensional array class, cupy.ndarray, and many functions on it.
 * [DeepSpeed MII](https://github.com/microsoft/DeepSpeed-MII) ![](https://img.shields.io/github/stars/microsoft/DeepSpeed-MII.svg?style=social) - DeepSpeed MII makes low-latency and high-throughput inference possible, powered by DeepSpeed.
+* [fastai](https://github.com/fastai/fastai) ![](https://img.shields.io/github/stars/fastai/fastai.svg?style=social) - fastai is a deep learning library which provides practitioners with high-level components that can quickly and easily provide state-of-the-art results in standard deep learning domains, and provides researchers with low-level components that can be mixed and matched to build new approaches.
 * [Flax](https://github.com/google/flax) ![](https://img.shields.io/github/stars/google/flax.svg?style=social) - A neural network library and ecosystem for JAX designed for flexibility.
+* [fastai](https://github.com/fastai/fastai) ![](https://img.shields.io/github/stars/fastai/fastai.svg?style=social) - fastai is a deep learning library which provides practitioners with high-level components that can quickly and easily provide state-of-the-art results in standard deep learning domains, and provides researchers with low-level components that can be mixed and matched to build new approaches.
+* [GPyTorch](https://github.com/cornellius-gp/gpytorch) ![](https://img.shields.io/github/stars/cornellius-gp/gpytorch.svg?style=social) - GPyTorch is a Gaussian process library implemented using PyTorch. GPyTorch is designed for creating scalable, flexible, and modular Gaussian process models with ease.
 * [H2O-3](https://github.com/h2oai/h2o-3) ![](https://img.shields.io/github/stars/h2oai/h2o-3.svg?style=social) - Fast scalable Machine Learning platform for smarter applications: Deep Learning, Gradient Boosting & XGBoost, Random Forest, Generalized Linear Modeling (Logistic Regression, Elastic Net), K-Means, PCA, Stacked Ensembles, Automatic Machine Learning (AutoML), etc..
+* [Ivy](https://github.com/unifyai/ivy) ![](https://img.shields.io/github/stars/unifyai/ivy.svg?style=social) - Ivy unifies the syntax and call signatures of existing frameworks.
```

Comment:
Similar to point on fastai this seems more like an interface / abstraction on top of other ML frameworks

Reply:
> Similar to point on fastai this seems more like an interface / abstraction on top of other ML frameworks

Good point! When I add this tool and I am hesitant about where to put it. I cannot find a single appropriate section to insert such an ML tool that specializes in ML framework interoperability. Maybe it is high time to create a new section like "Framework Interoperability" etc. What do you think? @axsaucedo

Reply:
Not sure about the interoperability as wrappers on top of ML libs are not productionisation frameworks, either way let's remove from PR and can merge

Reply:
> 

This tool aims to provide a transcompiler between frameworks and intends to minimize the gaps between platforms interoperability and indeed contribute the productionization of ML. @axsaucedo

Reply:
I'm not sure it seems it just converts a single api into multiple framework equivallent code, this doesn't seem computation optimization, but abstraction - at this stage i don't see this being in this section - let's remove to go ahead with PR

Reply:
Ok, I would remove it but keep a proposal then: https://github.com/EthicalML/awesome-production-machine-learning/issues/388